src/common/options/global.yaml.in

```diff
@@ -6502,3 +6502,12 @@ options:
   desc: Enables exception throwing instead of process abort on transaction submission error.
   default: false
   with_legacy: false
+- name: bluefs_wal_v2
+  type: bool
+  level: advanced
+  desc: Enables a faster write backend in BlueFS for WAL writes.
+  long_desc: By enabling this feature it will reduce ~50% the amount of fdatasync syscalls produced by WAL writes. This happens because we embed metadata
```

Comment:
Enabling this feature will reduce by 50% the number of fdatasync calls issued by WAL writes.